src/settings.py

```diff
@@ -6,6 +6,22 @@
 import botconfig
 
 LANGUAGE = 'en'
+
+## TLS settings
+# If SSL_CERTFP is supplied, the bot will attempt to verify that with the server. If not, then the
+# bot will attempt certificate verification, otherwise it will abort the connection.
+# You may specify the hash algorithm to use when verifying fingerprints. If unspecified, it will
+# attempt to autodetect the hash type of each fingerprint individually. (Limited to MD5, SHA1,
+# SHA224, SHA384, and SHA512).
+# Multiple fingerprints may be comma separated for networks with multipleservers. Check your Python
+# version for suported algorithms.
+# Syntax: SSL_CERTFP = "[HASH-ALGO:]fingerprint1,fingerprint2,fingerprint3"
+SSL_VERIFY = True
+SSL_CERTFILE = None # Client cert file to connect with in PEM format; can also contain keyfile.
+SSL_KEYFILE = None # Keyfile for the certfile in PEM format. if encrypted, password will prompt on the command line.
+SSL_CERTFP = None
+SSL_CIPHERS = None # Custom list of available ciphers in OpenSSL cipher list format. (<https://wiki.openssl.org/index.php/Manual:Ciphers(1)#CIPHER_LIST_FORMAT>)
```

Comment:
Depending on python version, the default when specifying None may not actually be secure. Better if we hardcode our own default here, that way we can update it to keep up with best practices instead of having every bot admin customize the list on their own.

Reply:
Planning to use Mozilla's "Modern" ciphersuite list.